otelcol/config.go

```diff
@@ -44,6 +46,11 @@ type Config struct {
 func (cfg *Config) Validate() error {
 	// Currently, there is no default receiver enabled.
 	// The configuration must specify at least one receiver to be valid.
+
+	if cfg == nil || reflect.ValueOf(cfg).Len() < 1 {
```

Comment:
Lets move this above the comments - those should be associate with the existing receiver check.

Reply:
Alright. I'm applying the changes right away